content/docs/forms.md

```diff
@@ -123,11 +123,11 @@ class EssayForm extends React.Component {
 }
 ```
 
-Notice that `this.state.value` is initialized in the constructor, so that the text area starts off with some text in it.
+Perhatikan bahwa `this.state.value` diinisialisasi di constructor, sehingga textarea mula-mula akan memiliki teks di dalamnya.
```

Comment:
should we translate constructor and textarea?

Reply:
just found out that there's no `konstruktor` in KBBI. so i think it's safe to leave it as *constructor*.

the `text area` in this context isn't referring to HTML tag `textarea` since it's separated with space. so IMO it has to be translated.

Reply:
@fajrizulfikar @vferdiansyah Yes, please leave it as is (*constructor*).

Reply:
ok, noted